### PR TITLE
Support for adding array parameters e.g. access_codes/list editing parameter access_code_ids

### PR DESCRIPTION
**Pull request by mastafit**
image

### CODE REVIEW

**Review by seveibar — CHANGES_REQUESTED** [review]
Can you try something more like this: https://chat.openai.com/share/82aa4e7b-43c4-4279-bb33-a347d85e4216

File: lib/interact-for-open-api-object.ts [hunk 1]
```diff
@@ -200,33 +200,44 @@ export const interactForOpenApiObject = async (
...
+      if (
+        (prop as any)?.items?.type === "string" &&
+        !(prop as any)?.items?.enum
+      ) {
+        let values = []
+        console.log(
+          `Enter multiple values for ${paramToEdit}, leave empty when done.`
+        )
+        while (true) {
+          const { value } = await prompts({
```
Comment (seveibar) [inline lib/interact-for-open-api-object.ts:213]: Create a subfunction in a seperate file called "interact for array"

**Comment by mastafit** [comment]
image
image

**Review by seveibar — APPROVED** [review]
nice

**Pull request merged**